kolibri/content/api.py

```diff
@@ -24,6 +26,28 @@ class ChannelMetadataCacheViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return models.ChannelMetadataCache.objects.all()
 
+    def destroy(self, request, pk=None):
```

Comment:
There is a device owner permission in the `Task` module that seems give all permissions. Can I use this?

```
class IsDeviceOwnerOnly(BasePermission):

    def has_permission(self, request, view):
        return request.user.is_superuser

    def has_object_permission(self, request, view, obj):
        return request.user.is_superuser

```